compiler/rustc_feature/src/active.rs

```diff
@@ -489,6 +489,8 @@ declare_features! (
     (active, plugin, "1.0.0", Some(29597), None),
     /// Allows exhaustive integer pattern matching on `usize` and `isize`.
     (active, precise_pointer_size_matching, "1.32.0", Some(56354), None),
+    /// Allows creating a `b'b'` `Literal`
+    (active, proc_macro_byte_character, "CURRENT_RUSTC_VERSION", Some(71358), None),
```

Comment:
I believe this list is only for compiler and language features. This entry shouldn't need to be here. The library stability attribute is sufficient.